### ISSUE TITLE
Misleading advice?

### ISSUE THREAD

**Issue by Sequential-circuits** [AlexeyAB/darknet#5653]
At the "How to improve object detection", it claims: "desirable that your training dataset include images with non-labeled objects that you do not want to detect - negative samples without bounded box (empty .txt files) - use as many images of negative samples as there are images with objects"

We did that: we included a similar number of unlabeled images with a product we did not want the system to recognize, and now it recognizes nothing.

It looks like a misleading advice

**Comment by VisionEp1** [comment]
the advice is not misleading. It is usually a good idea to mix negative samples into the data to prevent false negatives. 
However, you need to keep in mind that ai training is a non-deterministic process. 
Even settings which usually generate good results might generate worse results on a specific task or test set. 

Without knowing your exact configuration it's hard to tell where the issue is.

**Comment by Sequential-circuits** [comment]
I imagine it is not misleading, and the reason of my query.

We have 254 images of a product which we want to identify ("Original Energy" energy drink) and we put another 242 of another product we don´t want the computer to identify ("Old Spice" deodorant) 

Both products share some red parts and are rather similar in size, but I think the issue may be that the images of the deodorant are almost twice as big as the drink

**Comment by AlexeyAB** [comment]
What mAP did you get?
Show chart.png file
